ee/clickhouse/queries/breakdown_props.py

```diff
@@ -109,19 +130,29 @@ def _to_value_expression(
     breakdown_type: Optional[BREAKDOWN_TYPES],
     breakdown: Union[str, List[Union[str, int]], None],
     breakdown_group_type_index: Optional[GroupTypeIndex],
+    direct_on_events: bool = False,
 ) -> str:
     if breakdown_type == "person":
-        return get_single_or_multi_property_string_expr(breakdown, table="person", query_alias="value")
+        return get_single_or_multi_property_string_expr(
+            breakdown,
+            table="events" if direct_on_events else "person",
+            query_alias="value",
+            column="person_properties" if direct_on_events else "person_props",
```

Comment:
Hmm I wonder if there's a better way to do this column determination. Now, we have to spam some sort of conditional every time we call `get_single_or_multi_property_string_expr`

Reply:
I thought being explicit here is better than magic defaults or overriding based on a parameter 😅 . made things clearer

Reply:
.. and keeps this function dumb, which I also like

Reply:
Follow up TODO: `all` cohort doesn't seem to have tests / might blow up here. Also need to check if we still support this on the frontend

Reply:
fair let's roll with it then

Reply:
![image](https://user-images.githubusercontent.com/7115141/169560483-81ae177f-2a6b-42ee-a8a3-0bb947b66767.png)